chronicnetwork/assetlist.json

```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "Chronic Network",
+    "assets": [
+      {
+        "description": "The CHT coin is the governance token within the Chronic Chain network, aimed to serve the entire cannabis community through its payment network, supply chain and trust network.",
+        "denom_units": [
+          {
+            "denom": "ucht",
+            "exponent": 0
+          },
+          {
+            "denom": "cht",
+            "exponent": 6
+          }
+        ],
+        "base": "ucht",
+        "display": "cht",
+        "name": "Chronic Token",
+        "symbol": "CHT",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chronicnetwork/images/cht.png",
```

Comment:
please upload these images to the Cosmos Chain registry under chronicnetwork/images